README.md

```diff
@@ -58,6 +58,8 @@ Object with initial settings:
 
 * **callback**:   *function* that call when countdown end, default **null**;
```

Comment:
That is called once - when the timer is complete.

What I have added, will execute while the timer is running. If they specify the stepCount to be '5', then the callback will execute every 5 ticks.